### ISSUE TITLE
Write notebook comparing Albrow (2004) to our model

### ISSUE THREAD

**Issue by fbartolic** [fbartolic/hierarchical_single_lens_events#2]
Albrow's model: https://github.com/astrophpeter/AlbrowPredictive

Albrow trained his model on OGLE III EWS 2002 data and tested it on 2003 data, list of events is in the paper:
https://iopscience.iop.org/article/10.1086/383565/pdf

He doesn't compute any specific errors on the test set but rather just plots the histograms. We need to choose a metric such as the mean squared error (MSE), evaluate it for Albrow's MAP prediction and say the median prediction from the hierarchical model in either the data space or the parameter space.

**Comment by astrophpeter** [comment]
Is there an evaluation metric we can use that can be evaluated for both Albrow's point estimate predictions and our probabilistic ones? Perhaps some Monte Carlo expectation that uses the error bar on each future data point?

**Comment by fbartolic** [comment]
I don't think we need anything too fancy, a simple metric that demonstrates some improvement on the performance should suffice. Then we can focus on answering additional more interesting questions which can't be answered with the Albrow framework. 

Regarding the comparison to Albrow, for it to be a fair comparison, demonstrating superior predictive performance of a hierarchical model compared to empirical prior approach, we should implement the same model Albrow used to fit the training set in PyMC3. Otherwise you could argue that the improvement in predictive performance is due to a better individual model. I'll work on that in the next week or so.

**Comment by fbartolic** [comment]
Closing for now because a fair comparison turned out to be too tricky to make and the hierarchical advantage of the model is not obvious. Probably better to use simulated data in order to have control over all variables.
